packages/gateway-client/src/worker/p2p-fetch/lob-enc.ts

```diff
@@ -39,13 +40,16 @@ export const encode = function <J = Record<string, unknown>>(
         }
         // serialize raw json
         if (!Buffer.isBuffer(head)) {
+            head = {
```

Comment:
How necessary is it to add these changes? As opposed to doing it outside of this library?

Reply:
LOB encoding as is doesn't have any way of knowing where the body ends (and body can be arbitrarily long). I started doing it outside the library in various places, but missed it in a few places, or typo-ed it etc, so it made sense to me to add it here.